howfairis/mixins/checklist_mixin.py

```diff
@@ -1,16 +1,16 @@
 class ChecklistMixin:
 
     def check_checklist(self):
-        force_state = self.force_checklist
-        if force_state not in [True, False, None]:
-            raise ValueError("Unexpected configuration value for force_checklist.")
-        if isinstance(force_state, bool):
-            print("(5/5) checklist: force {0}".format(force_state))
-            return force_state
-        print("(5/5) checklist")
-        results = [
-            self.has_core_infrastructures_badge(),
-        ]
+        print("(5/5) checklist:")
+        reason = self.skip_checklist_checks_reason
+        if isinstance(reason, str):
+            if reason == "":
+                self._print_state(check_name="skipped (no reason provided)", state=True)
+            else:
+                self._print_state(check_name="skipped (reason: {0})".format(reason), state=True)
+            results = [True]
+        if reason is None:
+            results = [self.has_core_infrastructures_badge()]
```

Comment:
As config has already been passed through schema validation its either str or None. Can code be more 
```suggestion
        if reason is None:
            results = [self.has_core_infrastructures_badge()]
        else:
            if reason == "":
                self._print_state(check_name="skipped (no reason provided)", state=True)
            else:
                self._print_state(check_name="skipped (reason: {0})".format(reason), state=True)
            results = [True]
```

Reply:
updated in subsequent commits